scripts/validate.ts

```diff
@@ -43,6 +44,11 @@ export default async function (options: { verbose: boolean }, logger: logging.Lo
   error = await validateCommits({}, logger.createChild('validate-commits')) != 0
        || error;
 
+  logger.info('');
+  logger.info('Running DO_NOT' + '_SUBMIT validation...');
```

Comment:
Nit: here `DO_NOT' + '_SUBMIT` is used to (I assume) get around the check, but `DO_NOT${''}_SUBMIT` is used in `scripts/validate-do-not-submit.ts` instead. Would be helpful to have a comment explaining this and to make the workaround consistent.

Reply:
Done.